Performance/Performance112.md

```diff
@@ -16,10 +16,8 @@ We have partnered with BisectHosting this is a exciting step for us. All money e
 | Name | Known Incompatibilities | Description | Author | Performance Improvement | [Label](/README.md/#labels) |
 | --- | :---: | :---: | :---: | :---: | :---: |
 | [AE2 Unofficial Extended Life](https://www.curseforge.com/minecraft/mc-mods/ae2-extended-life) | Applied Energistics 2 | This is a Fork of [Applied Energistcs 2](https://www.curseforge.com/minecraft/mc-mods/applied-energistics-2) since 1.12.2 in no longer supported, This version of AE2 comes with many bugfixes and performance improvments | PrototypeTrousers | Both | none |
-| [AI Improvements](https://www.curseforge.com/minecraft/mc-mods/ai-improvements) | None | Simplified AI modification mod focused on performance and low-level modifications to AIs in the game | QueenOfMissiles | Server | none |
```

Comment:
Most of the mods that are removed had their functionality replaced by Universal Tweaks.

Reply:
Just because they have had their functionally replaced by another mod is not a reason to remove them, if the new mod is better and you have proof then add a note with the evidence (links for pictures, videos, and articles,)